lang/driver/src/info/collect.rs

```diff
@@ -131,11 +148,18 @@ impl CollectInfo for Data {
             // Add item
             let item = Item::Data(name.clone().id);
             collector.add_item(*span, item);
-            // Add info
-            let doc = doc.clone().map(|doc| doc.docs);
-            let info =
-                DataInfo { name: name.clone().id, doc, params: typ.params.print_to_string(None) };
-            collector.add_info(*span, info)
+
+            // Add hover info
+            let params = typ.params.print_to_string(None);
+            let mut content: Vec<MarkedString> = Vec::new();
+            content.push(MarkedString::String(format!("Data declaration: `{name}`")));
+            add_doc_comment(&mut content, doc.clone().map(|doc| doc.docs));
+            if !params.is_empty() {
+                content.push(MarkedString::String("---".to_owned()).to_owned());
+                content.push(MarkedString::String(format!("Parameters: `{}`", params)));
+                let hover_content = HoverContents::Array(content);
+                collector.add_hover(*span, hover_content);
```

Comment:
Hover content is only added when `!params.is_empty()`. This results in declarations without parameters not showing hover content.

Reply:
Good catch! Fixed in [cb09983](https://github.com/polarity-lang/polarity/pull/456/commits/cb099838273f933c8944cf5f3a90482e5881b2c8)